server/service/transport.go

```diff
@@ -23,6 +24,11 @@ func encodeResponse(ctx context.Context, w http.ResponseWriter, response interfa
 	// The has to happen first, if an error happens we'll redirect to an error
 	// page and the error will be logged
 	if page, ok := response.(htmlPage); ok {
+		// if the response is also an errorer and has an error, log it
+		if e, ok := response.(errorer); ok && e.error() != nil {
+			logging.WithErr(ctx, e.error())
+		}
```

Comment:
I've just checked and the `initiateSSOEndpoint` will log the error (calls `logged`). The issue here does not involve a fleet error. My mistake, this is not needed.

Reply:
initiate SSO will log it, but I don't think callback SSO will (because it implements the `page` interface).

Reply:
Ah nevermind, it still goes through the `logged` middleware, I'll remove this.